editor/Dockerfile

```diff
@@ -10,15 +10,17 @@ FROM $hubImage AS builder
 # Install editor
 ARG version
 ARG changeSet
-RUN unity-hub install --version "$version" --changeset "$changeSet" | grep 'Error' | exit $(wc -l)
+RUN unity-hub install --version "$version" --changeset "$changeSet" | tee install.log && grep 'Error' install.log | exit $(wc -l)
```

Comment:
Great idea to keep the install log in a file.

I would suggest using `/var/log/install-editor.log` and `/var/log/install-editor-module-${mod}.log` instead, so that there are no floating files in the root.